internal/server/get_lpa_details.go

```diff
@@ -0,0 +1,44 @@
+package server
+
+import (
+	"encoding/json"
+	"github.com/go-chi/chi/v5"
+	"github.com/ministryofjustice/opg-sirius-lpa-frontend/internal/sirius"
+	"net/http"
+
+	"github.com/ministryofjustice/opg-go-common/template"
+)
+
+type GetLpaDetailsClient interface {
+	CaseSummary(ctx sirius.Context, uid string) (sirius.CaseSummary, error)
+}
+
+type getLpaDetails struct {
+	XSRFToken string
+
+	LpaStoreData map[string]interface{}
+	CaseSummary  sirius.CaseSummary
+}
+
+func GetLpaDetails(client GetLpaDetailsClient, tmpl template.Template) Handler {
+	return func(w http.ResponseWriter, r *http.Request) error {
+		uid := chi.URLParam(r, "uid")
+		ctx := getContext(r)
+
+		caseSummary, err := client.CaseSummary(ctx, uid)
+
+		if err != nil {
+			return err
+		}
+
+		var lpaStoreData map[string]interface{}
+		_ = json.Unmarshal(caseSummary.DigitalLpa.LpaStoreData, &lpaStoreData)
```

Comment:
If you're not doing anything with the return value, I think it can be omitted altogether. Though I'm wondering if we should check whether there is an error here, just in case the JSON from the store is invalid.

Reply:
good shout! I'll add the error handling in